src/main/java/com/onevizion/scmdb/DdlGenerator.java

```diff
@@ -227,6 +229,7 @@ private String generateSequenceScripts(DbObject table) {
             }
             sequencesDdl.append(ddl);
         }
+        sequencesDdl.append("\r\n");
```

Comment:
`sequences`, `comments` and `triggers` doesn't required sorting?

Reply:
Done. 
Sorting takes place in an sql query (`extractTableDependentObjectsDdl()`)